keyboards/xd75/keymaps/atomic_style_jp/keymap.c

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2017 Wunder
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+ // 2019/03/04 takashiski
+ // NORTICE
+ // This layout is for Japanese Keyboard User.
+ // So there are logicalparing symbols and special keycode like a KC_RO, KC_HENK, KC_MHEN, KC_JYEN.
+ // 日本語キーボードとして認識させて使うことが前提なので、記号位置が英語キーボードとは異なります。
+ // また、日本語キーボードでないと認識されない特殊なキーコードが含まれています。
+
+#include "xd75.h"
```

Comment:
This should be replaced with: 
```suggestion
#include QMK_KEYBOARD_H
```